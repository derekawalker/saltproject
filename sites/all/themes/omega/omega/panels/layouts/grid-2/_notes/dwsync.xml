<?xml version="1.0" encoding="utf-8" ?><dwsync><file name="grid-2.css" server="ftp.beheros.com//public_html/" local="3547078414" remote="3520303200" Dst="-1" /><file name="grid-2.tpl.php" server="ftp.beheros.com//public_html/" local="3547078414" remote="3520303200" Dst="-1" /><file name="grid-2.inc" server="ftp.beheros.com//public_html/" local="3547078414" remote="3520303200" Dst="-1" /><file name="preview.png" server="ftp.beheros.com//public_html/" local="3547078414" remote="3520303200" Dst="-1" /></dwsync>