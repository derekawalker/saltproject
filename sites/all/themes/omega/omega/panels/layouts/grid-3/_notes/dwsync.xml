<?xml version="1.0" encoding="utf-8" ?><dwsync><file name="grid-3.inc" server="ftp.beheros.com//public_html/" local="3547078414" remote="3520303200" Dst="-1" /><file name="grid-3.tpl.php" server="ftp.beheros.com//public_html/" local="3547078414" remote="3520303200" Dst="-1" /><file name="preview.png" server="ftp.beheros.com//public_html/" local="3547078414" remote="3520303200" Dst="-1" /></dwsync>