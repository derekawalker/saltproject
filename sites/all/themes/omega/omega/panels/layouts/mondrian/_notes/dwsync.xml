<?xml version="1.0" encoding="utf-8" ?><dwsync><file name="mondrian.inc" server="ftp.beheros.com//public_html/" local="3547078414" remote="3520303200" Dst="-1" /><file name="mondrian.tpl.php" server="ftp.beheros.com//public_html/" local="3547078414" remote="3520303200" Dst="-1" /><file name="preview.png" server="ftp.beheros.com//public_html/" local="3547078414" remote="3520303200" Dst="-1" /></dwsync>