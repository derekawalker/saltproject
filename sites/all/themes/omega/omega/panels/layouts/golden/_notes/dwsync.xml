<?xml version="1.0" encoding="utf-8" ?><dwsync><file name="preview.png" server="ftp.beheros.com//public_html/" local="3547078414" remote="3520303200" Dst="-1" /><file name="golden.inc" server="ftp.beheros.com//public_html/" local="3547078414" remote="3520303200" Dst="-1" /><file name="golden.tpl.php" server="ftp.beheros.com//public_html/" local="3547078414" remote="3520303200" Dst="-1" /></dwsync>